### ISSUE TITLE
Multiple parameters?

### ISSUE THREAD

**Issue by cheezy** [redsquirrel/safariwatir#4]
I've written a simple dsl that a nice abstraction over Xwatir.  The dsl always passes a has to the method.  For example:

button(:id => 'some_id') OR button(:id => 'some_id', :index => 1)

This works fine on Watir and Firewatir but fails in Safariwatir.

**Comment by TreyE** [comment]
Yeah, currently this is well known bug.  It's something we need to put in a TODO section or something.

Right now, most methods don't support varargs, where the watir spec clearly identifies them as needing to do so.

For example, you're supposed to be able to find a div by "id" using simply:
browser.div("id"), or browser.div({ :id => "id" })
But right now watir only supports:
browser.div(:id, "id")
